pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,127 +21,15 @@ limitations under the License.
 package unversioned
 
 import (
-	"time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
 )
 
-func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
```

Comment:
I talked with @lavalamp and he said that he has a PR in-flight (that needs a rebase) that will separate registration from the generation. Maybe we can wait for that one?